src/pages/docs/getting-started/modal.md

```diff
@@ -30,10 +30,11 @@ modal. Let's update the component's template to do that then:
 /**
  * This renders the component's whole content.
  * Note: has to be called ".render".
- * @param body
- * @param header
  */
 &#123;template .render&#125;
+    &#123;@param body: string&#125;
+    &#123;@param header: string&#125;
+
```

Comment:
I don't think `&#123;` and `&#125;` is necessary here.
 - [ ] Can you try it with just `{` and `}`?